soc/ite/ec/common/vector.S

```diff
@@ -26,6 +26,31 @@ SECTION_FUNC(vectors, __start)
 
 	.option norvc;
 
+#ifdef CONFIG_SOC_IT8XXX2_JTAG_DEBUG_INTERFACE
+	/* Enable JTAG debug interface */
+	la t0, IT8XXX2_GCTRL_PMER3
+	lb t1, 0(t0)
+	ori t1, t1, 0x3
```

Comment:
Non-blocking comment: can you define macros for the PMER3 bits you're setting here?

Reply:
Done.